db/models.py

```diff
@@ -1 +1,30 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race, related_name="skill_set", on_delete=models.CASCADE
+    )
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255)
```

Comment:
Consider adding a `unique=True` constraint to the `nickname` field if each player should have a unique nickname. This aligns with the task requirement to handle unique constraints effectively.